expression/builtin.go

```diff
@@ -124,174 +123,6 @@ type functionClass interface {
 // BuiltinFunc is the function signature for builtin functions
 type BuiltinFunc func([]types.Datum, context.Context) (types.Datum, error)
 
-// Func is for a builtin function.
-type Func struct {
-	// F is the specific calling function.
-	F BuiltinFunc
-	// MinArgs is the minimal arguments needed,
-	MinArgs int
-	// MaxArgs is the maximal arguments needed, -1 for infinity.
-	MaxArgs int
-}
-
-// Funcs holds all registered builtin functions.
-var Funcs = map[string]Func{
```

Comment:
There are so many functions in this table, did you miss any one after the refactoring? How to confirm that?